### DISCUSSION TITLE
Deepin v23 Nightly no longer rounded corners to the dock

### DISCUSSION THREAD

**Discussion by ghost** [linuxdeepin/developer-center#3215] [Q&A | 问答板块] [answered]
Hi,
I noticed setting for rounded corners is no longer applied for the dock in v23 nightly, is it a temporary issue? Or can be changed to something else as development goes on 
![Screenshot_dde-desktop_20220602150448](https://user-images.githubusercontent.com/98664445/171625699-c7339981-fae9-4837-9fc4-67d287df6232.png)

**Comment by BLumia** [comment, marked as answer]
Yeah, it's a temporary issue, but we're not sure when it will be fixed. There will be some major change in the UI, so be aware it might even be completely broken during the v23 nightly updates ;p
